src/content/docs/distribute/snapcraft.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Snapcraft
+sidebar:
+  order: 1
+---
+# Prerequisites
+
+import { Tabs, TabItem, Card } from '@astrojs/starlight/components';
+
+**1. Install snap** 
+<Tabs>
+  <TabItem label="Debian">
+```sh
+  sudo apt install snapd
+```
+  </TabItem>
+<TabItem label="Arch">
+```sh
+sudo pacman -S --needed git base-devel
+  git clone https://aur.archlinux.org/snapd.git
+  cd snapd
+  makepkg -si
+  sudo systemctl enable --now snapd.socket
+  sudo systemctl start snapd.socket
+  sudo systemctl enable --now snapd.apparmor.service
+```
+  </TabItem>
+<TabItem label="Fedora">
+```sh
+  sudo dnf install snapd
+  sudo ln -s /var/lib/snapd/snap /snap # Enables classic snap support
+  #Reboot your system after doing this
+```
+  </TabItem>
+</Tabs>
+**2. Installing Core**
+<Tabs>
+  <TabItem label="V1">
+```sh
+  sudo snap install core20
+```
+  </TabItem>
+<TabItem label="V2">
+```sh
+  sudo snap install core22
+```
+  </TabItem>
+</Tabs>
+**3. Installing snapcraft**
+```sh
+  sudo snap install snapcraft --classic
+```
+1. Create an UbuntuOne account
+2. Go to the [Snapcraft](https://snapcraft.io) website and register an App name
+3. Create a snapcraft.yaml file in your projects root.
+4. Adjust the names in the snapcraft.yaml file
+
+```yaml
+name: app
+base: core22
+version: '2.0.4'
+summary: Your summary # 79 char long summary
+description: |
+  Your Description
+
+grade: stable
+confinement: strict
+
+apps:
+  app:
+    command: bin/app
+    desktop: usr/share/applications/app.desktop
+
+package-repositories:
+  - type: apt
+    components: [main]
+    suites: [noble]
+    key-id: 78E1918602959B9C59103100F1831DDAFC42E99D
+    url: http://ppa.launchpad.net/snappy-dev/snapcraft-daily/ubuntu
+
+parts:
+  prep:
+    plugin: dump
+    build-snaps:
+      - node/20/stable
+      - rustup/latest/stable
+    build-packages:
+      - libwebkit2gtk-4.1-dev
+      - build-essential 
+      - curl 
+      - wget 
+      - file 
+      - libxdo-dev 
+      - libssl-dev 
+      - libayatana-appindicator3-dev 
+      - librsvg2-dev
+      - dpkg
+    stage-packages:
+      - libwebkit2gtk-4.1-0
+      - libglu1-mesa
+      - libsoup-3.0-0
+      - freeglut3
+    source: .
+    override-pull: |
+      set -eu
+      craftctl default
+      rustup default nightly
+      npm install
+      npm run tauri build -- --bundles deb
+      dpkg -x src-tauri/target/release/bundle/deb/*.deb here
+      sed -i -e "s|Icon=app|Icon=/usr/share/icons/hicolor/32x32/apps/app.png|g" here/usr/share/applications/app.desktop
+      cp -r here/* .
+    organize:
+      usr/bin/app: bin/app
+```
+// TODO: ADD EXPLANATION
+
+# Explanation
+- The `name` variable defines the name of your app and is required to be set to the name that you have registered earlier.
+- The `base`variable defines which core you are using. For Tauri V1 you can use core20, but for Tauri v2 you must use core22
```

Comment:
same here then. Is core24 a thing already? (and does it work for tauri v2?)

Reply:
Core24 is a thing, and theoretical since I'm not using any plugins or extensions it should work, I can perform some testing if you want?

Reply:
If you have the time sure, if not then we can stick with 22 for now

Reply:
ok